tools/pythonpkg/tests/types/test_nested.py

```diff
@@ -0,0 +1,53 @@
+import duckdb
+
+
+class TestNested(object):
+
+    def test_lists(self, duckdb_cursor):
+        duckdb_conn = duckdb.connect()
+        result = duckdb_conn.execute("SELECT LIST_VALUE(1, 2, 3, 4) ").fetchall()
+        assert result == [([1, 2, 3, 4],)]
+
+        result = duckdb_conn.execute("SELECT LIST_VALUE() ").fetchall()
+        assert result == [([],)]
+
+        result = duckdb_conn.execute("SELECT LIST_VALUE(1, 2, 3, NULL) ").fetchall()
+        assert result == [([1, 2, 3, None],)]
+
+
+    def test_nested_lists(self, duckdb_cursor):
+        duckdb_conn = duckdb.connect()
+        result = duckdb_conn.execute("SELECT LIST_VALUE(LIST_VALUE(1, 2, 3, 4), LIST_VALUE(1, 2, 3, 4)) ").fetchall()
+        assert result == [([[1, 2, 3, 4], [1, 2, 3, 4]],)]
+
+        result = duckdb_conn.execute("SELECT LIST_VALUE(LIST_VALUE(1, 2, 3, 4), LIST_VALUE(1, 2, 3, NULL)) ").fetchall()
+        assert result == [([[1, 2, 3, 4], [1, 2, 3, None]],)]
+
+    def test_struct(self, duckdb_cursor):
+        duckdb_conn = duckdb.connect()
+        result = duckdb_conn.execute("SELECT STRUCT_PACK(a := 42, b := 43)").fetchall()
```

Comment:
missing tests with empty structs and lists here, and structs/list that are NULL or only contain NULL

Reply:
There was already a test for empty lists, I didn't add the one for empty structs since is not possible to create them (i.e., struct_pack throws an error "Can't pack nothing into a struct".

I've added tests for structs and lists that only contain nulls, and a struct containing an empty list.